src/components/dls/Separator/Separator.module.scss

```diff
@@ -1,5 +1,44 @@
 $separator-size: 1px;
 
+.faded {
```

Comment:
Changing the separator styling will result in visual changes to any place we are using this component:

<img width="519" alt="Screen Shot 2021-12-20 at 18 54 19" src="https://user-images.githubusercontent.com/15169499/146763431-b91f3be3-2a75-4e88-b029-27b17b106842.png">

these values should be passed from the parent component wanting this change as the value of `className` prop. Please see my comment above on `TranslationViewCell.tsx`.